packages/esm-patient-chart-app/src/patient-flags/patient-flags.scss

```diff
@@ -0,0 +1,58 @@
+@use '@carbon/styles/scss/spacing';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.flagIcon{
+    font-size: medium;
+}
+
+.flagTag {
+  padding: 0 0.5rem;
+  span {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.infoFlagTag {
+  padding: 0 0.5rem;
+  background-color: #ffd9be;
```

Comment:
Do we have a variable for this?

Reply:
No we don't, I think I will push another PR to add it to core. It is a new orange color, no even in carbon.